app/pages/organization/stats/components/BarChart/components/Parameters.jsx

```diff
@@ -130,15 +167,23 @@ function Parameters({
           options={maxDateRange}
           selectFor="rangeMax"
         />
-        <button
+      </StyledInputsContainer>
+      <StyledResetContainer>
+        <StyledResetButton
           onClick={() => handleReset()}
           type="button"
         >
           <Translate
             content="organization.stats.reset"
           />
-        </button>
-      </StyledInputsContainer>
+        </StyledResetButton>
+        {binBy === 'hour' ? (
```

Comment:
You could do `binBy === 'hour' && ( <Translate` here to avoid the unnecessary ternary, but the ternary still works fine.